common/text.c

```diff
@@ -1,9 +1,17 @@
 #include "text.h"
 
-//TODO: Update this once libnx supports settings get-language.
-
 static int s_textLang = /*CFG_LANGUAGE_EN*/1;
 
+void textInit(void) {   
```

Comment:
There's zero error checking here.

Should call setExit at the end of this func too.